implementations/rust-boon/src/main.rs

```diff
@@ -6,6 +6,7 @@ use serde_json::{json, Map, Value};
 fn main() -> Result<(), Box<dyn Error>> {
     let mut started = false;
     let mut draft = None;
+    let osinfo = os_info::get();
```

Comment:
Same here, I think you're doing this on every request as opposed to just where you need it.

Reply:
Yeah sorry!